earth_enterprise/src/fusion/autoingest/MiscConfigStorage.idl

```diff
@@ -34,6 +34,9 @@ class MiscConfigStorage {
   uint64       MaxVersionCacheMemorySize = uint64(2415919104);
 
   bool         LimitMemoryUtilization = false;
+
+  // The percentage of cache size that the dirty map size must meet for cache to not be pruned.
```

Comment:
This comment is worded a bit awkwardly. I'd suggest something like "The cache will not be pruned if the dirty map is more than this fraction of the cache."